test/Microsoft.AspNet.Mvc.Core.Test/ParameterBinding/ControllerActionArgumentBinderTests.cs

```diff
@@ -470,8 +610,13 @@ private class TestController
         {
             public string UnmarkedProperty { get; set; }
 
-            [NonValueProviderBinderMetadata]
-            public string NonValueBinderMarkedProperty { get; set; }
+            public string[] ArrayProperty { get; } = new string[] { "goodbye" };
+
+            public ICollection<string> CollectionValueBinderMarkedProperty { get; } = new List<string> { "goodbye" };
```

Comment:
These properties without any attribute makes it inconsistent with the way this class was written. 
Now there are few properties with attributes and rest of them without attributes. 
Suggest remove the attributes from the properties of this class as they are not discovered based on attributes but are directly added to the `BoundProperties` in individual tests, and add a comment on `TestController` mentioning the reason why none of the properties need attributes.